variables.tf

```diff
@@ -50,3 +51,9 @@ variable "buckets_list" {
     error_message = "Bucket names can only contain lowercase letters, numeric characters, dashes (-), underscores (_). Bucket names must start and end with a number or letter. Bucket names must contain 6-58 characters (5 chars are reserved for random suffix). We do not allow dots (.) even if they are allowed as per documentation https://cloud.google.com/storage/docs/buckets#naming"
   }
 }
+
+variable "google_tags_value_list" {
```

Comment:
Why we don't put this at in the object inside the `buckets_list`? I think we could define different tag values per bucket, and not the same tag values for all the `bucket_list`.